static/app/views/nav/primary/onboarding.tsx

```diff
@@ -90,7 +90,7 @@ function OnboardingItem({
               textCss={() => css`
                 font-size: ${isMobile ? theme.fontSizeExtraSmall : theme.fontSizeSmall};
                 font-weight: ${theme.fontWeightBold};
-                color: ${theme.purple400};
+                color: ${theme.tokens.content.accent};
```

Comment:
@Jesse-Box now this is blue and not purple. is that what we want?

![image](https://github.com/user-attachments/assets/9f19c845-bb66-4a48-a630-4d3128f06006)

Reply:
Wasn't intentional, keep it purple :)